x-pack/filebeat/input/httpjson/config.go

```diff
@@ -62,9 +63,7 @@ type RateLimit struct {
 
 func (c *config) Validate() error {
 	switch strings.ToUpper(c.HTTPMethod) {
-	case "GET":
-		break
-	case "POST":
+	case "GET", "POST":
```

Comment:
Could you add validation for any raw json strings in your config? Example from my http_endpoint input:

```
func (c *config) Validate() error {
	if !json.Valid([]byte(c.ResponseBody)) {
		return errors.New("response_body must be valid JSON")
	}

	return nil
}
```

Reply:
Added it in the places where it is expected: https://github.com/elastic/beats/pull/18892/files#diff-e3d27d33f878a5fb0a720224c80ad7aaR135 and https://github.com/elastic/beats/pull/18892/files#diff-e3d27d33f878a5fb0a720224c80ad7aaR171